src/Lumi/Components/Form.purs

```diff
@@ -383,6 +405,43 @@ switch = formBuilder_ \{ readonly } b onChange ->
            , onChange = Events.handler (stopPropagation >>> targetChecked) (traverse_ onChange)
            }
 
+-- | A `checkbox` is an editor for booleans which displays checked or not checked.
+-- | can also accept JSX to display text to the right of the checkbox
+checkbox
+  :: Maybe JSX
+  -> forall props
+  . FormBuilder { readonly :: Boolean | props }
+  Boolean
+  Boolean
+checkbox description =
+  formBuilder_ \{ readonly } value onChange ->
+    Input.label
+      { style: R.css
+          { flexDirection: "row"
+          , alignSelf: "stretch"
+          }
+      , for: null
+      , children:
+          [ row
+              { style: R.css { alignItems: "center" }
```

Comment:
I believe we don't want vertical centering here, do we? I'd say probably `baseline` or `flex-start`.

Reply:
Actually according to the designs we want the description title to be perfectly centered to the checkbox and then the description subtitle sitting below both (the title + checkbox). This can potentially look weird if someone sets up a checkbox with just a subtitle (and no title) as it'll be sitting below, but I don't see why that case would ever occur (why would you just have a subtitle and no title? additionally, this is taking JSX so if we do want a descriptive title with a "subtext" appearance we can pass that in)... let me know what you think and if there might be a better approach.

Reply:
I feel like `baseline` is what we want then, no? The first line of whatever JSX is passed in will be aligned with the checkbox. We can adjust the "perfect centering" with margins/padding I believe.